likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/utils/LMFeedStyleTransformer.kt

```diff
@@ -27,4 +32,45 @@ object LMFeedStyleTransformer {
     @JvmStatic
     var likesFragmentViewStyle: LMFeedLikesFragmentViewStyle =
         LMFeedLikesFragmentViewStyle.Builder().build()
+
+    @JvmStatic
+    var reportFragmentViewStyle: LMFeedReportFragmentViewStyle =
+        LMFeedReportFragmentViewStyle.Builder().build()
+
+    @JvmStatic
+    var adminDeleteDialogFragmentStyle: LMFeedAdminDeleteDialogFragmentStyle =
+        LMFeedAdminDeleteDialogFragmentStyle.Builder().build()
+
+    @JvmStatic
+    var selfDeleteDialogFragmentStyle: LMFeedAlertDialogStyle =
+        LMFeedAlertDialogStyle.Builder()
+            .alertSubtitleText(
+                LMFeedTextStyle.Builder()
+                    .textColor(R.color.lm_feed_grey)
+                    .textSize(R.dimen.lm_feed_text_medium)
+                    .typeface(Typeface.NORMAL)
+                    .fontResource(R.font.lm_feed_roboto)
+                    .build()
+            )
+            .alertNegativeButtonStyle(
+                LMFeedTextStyle.Builder()
+                    .textAllCaps(true)
+                    .textColor(R.color.lm_feed_black_40)
+                    .textSize(R.dimen.lm_feed_text_small)
+                    .typeface(Typeface.NORMAL)
+                    .fontResource(R.font.lm_feed_roboto_medium)
+                    .build()
+            )
+            .alertPositiveButtonStyle(
+                LMFeedTextStyle.Builder()
+                    .textAllCaps(true)
+                    .textColor(R.color.lm_feed_black_20)
+                    .textSize(R.dimen.lm_feed_text_small)
+                    .typeface(Typeface.NORMAL)
+                    .fontResource(R.font.lm_feed_roboto_medium)
+                    .build()
+            )
+            .alertBoxElevation(R.dimen.lm_feed_elevation_small)
+            .alertBoxCornerRadius(R.dimen.lm_feed_corner_radius_small)
+            .build()
```

Comment:
why were are building style here.

Reply:
initially I was trying to use an alert dialog directly without creating a fragment, that's why I updated the style here only. changing it to a fragment style now